### DISCUSSION TITLE
Problemi con attivazione ambiente di Test ANSC Comune di Ardauli

### DISCUSSION THREAD

**Discussion by SattaMem** [italia/ansc#1163] [Supporto Webapp]
Buongiorno,
stiamo avendo delle difficoltà  nell'attivare l'ambiente di test ANSC del Comune di Ardauli.
Abbiamo inviato email a supporto-ansc@sogei.it per la creazione dello stesso, ma è stato risposto quanto:

Gent.le Comune,
il vostro amministratore locale della sicurezza (ALS) può accedere con la
propria identità digitale allla console di sicurezza di test al link
https://anscwebpre.anpr.interno.it/ansc/login/anpr
e censire le utenze necessarie
Saluti
Assistenza ANSC

Accedendo però con lo spid dell'operatore di ALS una volta digitate le credenziali compare errore generico di login e non riusciamo ad andare avanti.

Abbiamo tentato anche la seconda opzione di censire nell'ambiente test ANSC gli utenti ma non ci permette di accedere al link
https://webtest.anpr.interno.it/

Siamo pertanto bloccati.

Si richiede supporto urgente.

Saluti

**Comment by rvinzani** [comment]
Buongiorno, probabilmente non avete il certificato di postazione di test. Lo invieremo in allegato rispondendo alla mail che avete inviato alla casella supporto-ansc. Saluti

**Comment by rvinzani** [comment]
Il certificato è stato inviato. Saluti

**Comment by SattaMem** [comment]
Buongiorno, certificato installato, lo riconosce, ma compare sempre l'errore login error

Abbiamo provato entrambi i link con credenziali spid utente als:

https://webtest.anpr.interno.it/anprconsole/home.xhtml

https://anscwebpre.anpr.interno.it/ansc/login/anpr#

vedi allegato
[errore.docx](https://github.com/user-attachments/files/17830788/errore.docx)

**Reply by rvinzani** [reply]
Dovete selezionare il certificato CA postazione svil

**Reply by SattaMem** [reply]
![Cattura](https://github.com/user-attachments/assets/776910ca-3396-4ebb-a10f-aedafb143dd0)
Salve, selezioniamo quello, ma subito dopo compare login error

**Comment by rvinzani** [comment]
Con quale utenza provate ad entrare? Magari mi indica le prime lettere del CF. Grazie

**Reply by SattaMem** [reply]
LOIDRH

**Reply by rvinzani** [reply]
Poichè non risulta censito come ALS in ambiente di test può per favore inviare a supporto-ansc@sogei.it il suo CF ed indirizzo email così lo censisco

**Reply by SattaMem** [reply]
Buongiorno, è possibile ricevere anche il certificato Server dell'ambiente test? E' necessario per poter svolgere le integrazioni con il software.
Rimaniamo in attesa.

**Comment by SattaMem** [comment]
Buongiorno, siamo bloccati, potremo avere supporto in merito?
